### ISSUE TITLE
Enhance font thickness

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#276]
*On GitLab by @miseran on Jun 9, 2021, 10:54*

---

On low DPI displays, some fonts (like computer modern) can be very thin and hard to read. This is especially the case when using a dark recolor. It would be great if zathura could provide a feature to enhance legibility in that case.

I don't know if there is a way to improve this on the side of poppler / mupdf through font hinting or some such. If not, I suggest a setting to apply gamma correction to the output. I hacked together a proof of concept that applies a gamma correction of 1.5 before recoloring, see the following screenshot. In my opinon, the gamma corrected output (the lower screenshot) is much easier on the eyes. If there is interest, I can try implementing gamma correction as a proper setting.

![zathura-gamma](https://git.pwmt.org/pwmt/zathura/uploads/70c517bb02e7f68a8ed18f039bb42955/zathura-gamma.png)

**Comment by MithicSpirit** [comment]
I agree with the idea of applying a gamma correction. Xdvi allows customizable gamma, and this makes reverse video much more legible.
